ios/brave-ios/Sources/BraveVPN/Components/Region/BraveVPNRegionDetailsView.swift

```diff
@@ -0,0 +1,191 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import BraveStrings
+import GuardianConnect
+import SwiftUI
+
+struct BraveRegionDetailsView: View {
+
+  @State
+  private var isAutoSelectEnabled: Bool
+
+  @State
+  private var isLoading = false
+
+  @State
+  private var isConfirmationPresented = false
+
+  @State
+  private var isShowingChangeRegionAlert = false
+
+  @ObservedObject
+  private var cityRegionDetail: VPNCityRegionDetail
+
+  @State
+  private var regionModificationTimer: Timer?
+
+  @State
+  private var cityRegions: [GRDRegion]
+
+  public init(
+    countryRegion: GRDRegion?,
+    with cityRegions: [GRDRegion] = [],
+    isAutoSelectEnabled: Bool = true
+  ) {
+    self.isAutoSelectEnabled = isAutoSelectEnabled
+    self.cityRegions = cityRegions
+
+    var regions: [BraveVPNCityRegion] = []
+
+    for cityRegion in cityRegions {
+      regions.append(
+        BraveVPNCityRegion(displayName: cityRegion.displayName, regionName: cityRegion.regionName)
+      )
+    }
+
+    cityRegionDetail = VPNCityRegionDetail(
+      countryName: countryRegion?.country ?? "",
+      countryISOCode: countryRegion?.countryISOCode ?? "",
+      cityRegions: regions
+    )
+  }
+
+  var body: some View {
+    ZStack {
+      List {
+        Section(header: Text(Strings.VPN.availableServerTitle)) {
+          ForEach(cityRegionDetail.cityRegions) { server in
+            cityRegionItem(at: 0, region: server)
+          }
+        }
+        .listRowBackground(Color(braveSystemName: .containerBackgroundMobile))
+      }
+      .opacity(isLoading ? 0.5 : 1.0)
+
+      if isLoading {
+        BraveVPNRegionLoadingIndicatorView()
+          .transition(.opacity)
+          .zIndex(1)
+      }
+    }
```

Comment:
Avoid `ZStack`, apply `overlay` to the `List` itself

Reply:
Fixed at https://github.com/brave/brave-core/pull/24198/commits/db1d988f0db57643c18b4dd3cbca47d5c3adc98f